specification/compute/resource-manager/Microsoft.Compute/stable/2018-04-01/compute.json

```diff
@@ -337,6 +337,270 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/proximityPlacementGroups/{proximityPlacementGroupName}": {
+      "put": {
+        "tags": [
+          "ProximityPlacementGroups"
+        ],
+        "operationId": "ProximityPlacementGroups_CreateOrUpdate",
+        "description": "Create or update a proximity placement group.",
+        "parameters": [
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "proximityPlacementGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the proximity placement group."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ProximityPlacementGroup"
+            },
+            "description": "Parameters supplied to the Create Proximity Placement Group operation."
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/ProximityPlacementGroup"
+            }
+          },
+          "201": {
+            "description": "Created",
+            "schema": {
+              "$ref": "#/definitions/ProximityPlacementGroup"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Create a proximity placement group.": {
+            "$ref": "./examples/CreateAProximityPlacementGroup.json"
+          }
+        }
```

Comment:
Is createOrUpdate a long running operation? 
Please double check in server side and confirm, If yes apply relevant extension [example for PUT](https://github.com/Azure/azure-rest-api-specs/blob/f914e54203326a215b1bc66ad5e8f1a1aa7d1e75/specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json#L558).

Reply:
They are not long running operation, final response is returned in sync path